lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -79,6 +79,8 @@ def autocorrect(node)
         private
 
         def find_usage(node, &block)
+          return if node.nil?
```

Comment:
IMO the return should be in the `on_block`, before even trying to find the usage inside the body

Reply:
Thank you – I was wondering about exactly that.